config/routes.rb

```diff
@@ -1,6 +1,7 @@
 Rails.application.routes.draw do
-  # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
-
-  # Defines the root path route ("/")
-  # root "articles#index"
+  root 'users#index'
+  resources :users do
+    resources :posts
+  end
```

Comment:
- [ ] Please do not expose unnecessary endpoints in your `routes.rb`, make sure only routes needed for users and posts are created 👍🏿 

```
  resources :users, only: [:index, :show] do
    resources :posts, only: [:index, :show]
  end
```

Reply:
Thanks for your review!